source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
+Navigating the catalog
 ======================
 
-OpenDataSoft lets you find datasets with a rich search based experience. The catalog is often accessible via the **Explore** or **Datasets** link on the header menu. It allows to filter through all the portal's datasets by keyword, publisher, date or theme, among others.
+The dataset catalog of an OpenDataSoft portal is accessible from the front office for all users allowed on the portal (i.e. for open portals it could be anyone wheareas for private portals it should be only authenticated users).
 
-.. image:: images/explore_all.png
-   :alt: Explore catalog
+The catalog, depending on the portal, can be the default landing page of that portal. However, if that portal has another page as default landing page (e.g. a homepage), a link to the catalog should be in the navigation bar at the very top of the front office. Since this navigation bar is fully customizable, the label of the link to access the catalog can be different from one portal to another (very often, "Data", "Catalog", "Explore" or "Datasets" used as labels).
 
-Navigating in the catalog
--------------------------
+The structure of a dataset catalog however remains the same no matter the portal.
 
-The main part of the catalog shows a series of cards representing each dataset available on the platform. On each card, the dataset's metadata are displayed, along with quick links to visualize the data.
+.. image:: images/catalog_fullinterface.png
+   :alt: Full interface of the catalog, with both sides
 
-.. image:: images/explore_card.png
-   :alt: Dataset card
+On the left side (1), are displayed:
 
-On this example, the card is split in two parts. On the left part, there is a theme pictogram, the title, a description, the language and license, keywords and the original publisher. On the right part, there are links to access the data table, a map and analysis tools, and to export or access the API of the dataset.
+- the number of datasets currently displayed,
+- a catalog sort option to change the order of the datasets,
+- a variety of catalog filters to find specific datasets,
+- different possibilities to download the catalog.
 
-Like most of the catalog's elements, this card is fully customizable.
+On the right side (2), are displayed the datasets. Each dataset is represented by a rectangle called a "catalog card". On each card, the dataset's main metadata are displayed, along with quick links to visualize the data.
+
+Sorting the catalog
+-------------------
+
+By default, a dataset catalog is sorted by last modification date (i.e. the most recently modified datasets are displayed in first position, at the top of the catalog).
+
+It is possible to change in which order the catalog cards are displayed with the catalog sorting option below the number of datasets, on the left side of the interface.
+
+.. image:: images/catalog_sort.png
+   :alt: Catalog sorting option
+
+The catalog can be sorted by 10 different methods:
```

Comment:
I find this list of "10 different methods" a bit frightening at first read. What about only documenting the 5 main sorting criteria and stating that for each its possible to sort by increasing order or decreasing order. This way,  the reader is really shown with the criteria definition and the rest is a matter of UI.

So, would end up with something like:

The catalog can be sorted by different criteria:
- Title alphabetical order
- Modification date
- Number of records
- Number of downloads
- Popularity

For each of these criteria, it is possible to sort by increasing or decreasing order.

This approach would also make impacts on the documentation less important as the UI evolves.

Reply:
Yes you're right, that's a very good way to simplify this section of the documentation. I will change that list!